Dockerfile.build-base

```diff
@@ -0,0 +1,13 @@
+FROM mesosphere/mesos:0.22.1-1.0.ubuntu1404
```

Comment:
Currently, no. It would potentially make sense to use this for building the assembly of the official image as well but as far as I know two phase builds are not supported on dockerhub,